node/trieIterators/factory/sovereignDelegatedListHandlerFactory.go

```diff
@@ -0,0 +1,24 @@
+package factory
+
+import (
+	"github.com/multiversx/mx-chain-go/node/external"
+	"github.com/multiversx/mx-chain-go/node/trieIterators"
+)
+
+type sovereignDelegatedListHandlerFactory struct {
+}
+
+// NewSovereignDelegatedListHandlerFactory create a new sovereign delegated list handler
+func NewSovereignDelegatedListHandlerFactory() *sovereignDelegatedListHandlerFactory {
+	return &sovereignDelegatedListHandlerFactory{}
+}
+
+// CreateDelegatedListHandler will create a new instance of DirectStakedListHandler
```

Comment:
Perhaps we should be consistent with verb tenses here(e.g. in constructor creates, but here: will create)?

also, creates a direct staked list processor for sovereign